Recrutify/Recrutify.Services/Validators/BulkUpdateStatusReasonCandidatsValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Linq;
+using FluentValidation;
+using Recrutify.Services.DTOs;
+
+namespace Recrutify.Services.Validators
+{
+    public class BulkUpdateStatusReasonCandidatsValidator : AbstractValidator<BulkUpdateStatusDTO>
+    {
+        public BulkUpdateStatusReasonCandidatsValidator()
+        {
+            RuleFor(x => x.CandidatesIds)
+                .NotNull()
```

Comment:
return candidatesexising method and mustAsync rule

Reply:
done